turbo.json

```
@@ -20,6 +20,15 @@
       "outputs": ["dist/**"],
       "dependsOn": ["^build", "compile"]
     },
+    "build-metachain": {
+      "inputs": ["Cargo.toml", "Cargo.lock", "rustfmt.toml", "meta/**"],
+      "outputs": ["target/**"]
+    },
+    "build-docker": {
+      "inputs": ["Dockerfile"],
+      "outputs": [],
+      "dependsOn": ["^build"]
+    },
```

```suggestion
    "build:cargo": {
      "inputs": ["Cargo.toml", "Cargo.lock", "rustfmt.toml", "meta/**"],
      "outputs": ["target/**"]
    },
    "build:docker": {
      "inputs": ["Dockerfile"],
      "outputs": [],
      "dependsOn": ["^build:cargo"]
    },
```

Possibly we need something like this.